src/docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -26,7 +26,7 @@ To create a metric alert you first need to choose a metric type. For some alert
 
 #### Performance
 
-For performance alerts, [metric-backed](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your alert that's not metrics-compatible, then your alert will use indexed events, or event-based data, and a message saying this will be displayed.
+For performance alerts, the data is based on [processed transaction events](/product/sentry-basics/search/searchable-properties/#processed-event-properties) data is the default. If you make a selection in configuring your alert that's not metrics-compatible, then your alert will use indexed events, or event-based data, and a message saying this will be displayed.
```

Comment:
```suggestion
For performance alerts, data based on [processed transaction events](/product/sentry-basics/search/searchable-properties/#processed-event-properties) is the default. If you make a selection in configuring your alert that's not compatible with processed event data, then your alert will use indexed transaction events, or event-based data, and a message saying this will be displayed.
```